components/homme/src/theta-l_kokkos/cxx/HyperviscosityFunctorImpl.cpp

```diff
@@ -118,9 +118,14 @@ void HyperviscosityFunctorImpl::init_params(const SimulationParams& params)
   m_eos.init(params.theta_hydrostatic_mode,m_hvcoord);
 
 #ifdef HOMMEXX_BFB_TESTING
-  m_process_nh_vars = true;
+  m_process_nh_vars = 1;
 #else
-  m_process_nh_vars = !params.theta_hydrostatic_mode;
+  //m_process_nh_vars = !params.theta_hydrostatic_mode;
+  if (params.theta_hydrostatic_mode){
```

Comment:
You can just do
```c++
m_process_nh_vars = static_cast<int>(params.theta_hydrostatic_mode);
```
Faster to read, and makes your intent of simply changing the type clearer.